tasmota/language/bg_BG.h

```diff
@@ -622,6 +622,11 @@
 #define D_SENSOR_MAX31855_CS   "MX31855 CS"
 #define D_SENSOR_MAX31855_CLK  "MX31855 CLK"
 #define D_SENSOR_MAX31855_DO   "MX31855 DO"
+#define D_SENSOR_MAX31865_CS1  "MX31865 CS1"
+#define D_SENSOR_MAX31865_CS2  "MX31865 CS2"
+#define D_SENSOR_MAX31865_CS3  "MX31865 CS3"
+#define D_SENSOR_MAX31865_CS4  "MX31865 CS4"
+#define D_SENSOR_MAX31865_CS5  "MX31865 CS5"
```

Comment:
Don't do this. Look at how buttons or relays are implemented. Waste of translation space.